rules/combo/recon/nodejs.yara

```diff
@@ -1,14 +1,53 @@
 
-rule npm_sysinfoexfil : high {
+rule nodejs_sysinfoexfil : high {
   meta:
     description = "may gather and exfiltrate system information"
     hash_2023_botbait = "1b92cb3d4b562d0eb05c3b2f998e334273ce9b491bc534d73bcd0b4952ce58d2"
   strings:
     $proc1 = "process.platform"
     $proc2 = "process.arch"
     $proc3 = "process.versions"
-    $h = "http.request"
+	$request_http = /require\([\"\']https{0,1}[\"\']\)/
     $post = "POST"
   condition:
-    filesize < 33554432 and $h and $post and any of ($proc*)
+    filesize < 16KB and $request_http and $post and any of ($proc*)
+}
+
+rule nodejs_phone_home : high {
```

Comment:
Does this need to be private or does that not matter all that much?

Reply:
nope, I still want this rule to show if it matches, even if nothing that relies on it does.